File: documentation-examples-scala/src/scala/dev/typr/foundations/docs/oracle/StringTypes.scala
package dev.typr.foundations.docs.oracle
import dev.typr.foundationssc.*
import dev.typr.foundationssc.data.*



@SuppressWarnings(Array("unused"))
object StringTypes:
  //start
  val varcharType: OracleType[String] = OracleTypes.varchar2
  val varchar100: OracleType[String] = OracleTypes.varchar2(100)
  val charType: OracleType[String] = OracleTypes.char_(10)
  val nvarcharType: OracleType[String] = OracleTypes.nvarchar2(100)
  //stop

File: documentation-examples-java/src/java/dev/typr/foundations/docs/oracle/StringTypes.java
package dev.typr.foundations.docs.oracle;

import dev.typr.foundations.OracleType;
import dev.typr.foundations.OracleTypes;

@SuppressWarnings("unused")
public class StringTypes {
  // start
  OracleType<String> varcharType = OracleTypes.varchar2;
  OracleType<String> varchar100 = OracleTypes.varchar2(100);
  OracleType<String> charType = OracleTypes.char_(10);
  OracleType<String> nvarcharType = OracleTypes.nvarchar2(100);
  // stop
}
